airbyte-integrations/connectors/source-mssql/metadata.yaml

```diff
@@ -18,20 +18,18 @@ data:
   name: Microsoft SQL Server (MSSQL)
   registries:
     cloud:
-      # CI pipeline is broken for mssql
-      dockerImageTag: 3.6.1
       enabled: true
     oss:
-      # CI pipeline is broken for mssql
-      dockerImageTag: 3.6.1
       enabled: true
-  releaseStage: alpha
+  releaseStage: generally_available
   supportLevel: community
   tags:
     - language:java
-    - language:python
   releases:
     breakingChanges:
+      4.0.0:
+        message: "Certified source mssql connector."
+        upgradeDeadline: "2024-04-07"
```

Comment:
should we be using scopedImpact? https://docs.airbyte.com/connector-development/connector-metadata-file/#scopedimpact

Reply:
I don't think `scopedImpact` is applicable here. 
There's no stream name that is breaking